cypress/e2e/calendar/add-hours-of-operation.cy.js

```diff
@@ -0,0 +1,122 @@
+import { Select } from '@interactors/html';
+import {
+  MultiColumnListCell, MultiColumnListRow, TimeDropdown, Pane, Button, IconButton, TextField, Accordion,
+  including, Link, matching
+} from '../../../interactors';
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+const addHoursOfOperationData = calendarFixtures.data.addHoursOfOperation;
+const addHoursOfOperationExpectedUIValues = calendarFixtures.expectedUIValues.addHoursOfOperation;
+
+
+describe('Add new hours of operation for service point', () => {
+  let testCalendarResponse;
+  before(() => {
+    // login and open calendar settings
+    cy.openCalendarSettings(false);
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point and calendar
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+      cy.openCalendarSettings();
+    });
+  });
+
+
+  after(() => {
+    // delete test calendar
+    cy.deleteCalendar(testCalendarResponse.id);
+  });
+
+
+  it('adds new hours of operation for service point', () => {
```

Comment:
`it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {`